app/services/get-overview.js

```diff
@@ -16,18 +18,34 @@ module.exports = function (id, organisationLevel) {
   }
 
   return overviewPromise.then(function (results) {
-    if (results.length !== undefined) {
-      results.forEach(function (result) {
-        result.capacityPercentage = (result.totalPoints / result.availablePoints) * 100
-      })
-      result.overviewDetails = results
-    } else {
-      var capacityPercentage = (results.totalPoints / results.availablePoints) * 100
-      result.overviewDetails = Object.assign({}, results, {capacity: capacityPercentage})
-    }
+    result.overviewDetails = calculateCapacityPercentage(results)
     result.breadcrumbs = getBreadcrumbs(id, organisationLevel)
     result.title = result.breadcrumbs[0].title
     result.subTitle = organisationalUnitType.displayText
     return result
   })
 }
+
+var calculateCapacityPercentage = function (results) {
```

Comment:
Don't think `calculateCapacityPercentage` should be also calling `getContractedHours`. Possibly a rename? Or take contracted hours out of this function